src/sp_demon.c

```diff
@@ -506,7 +506,7 @@ float DemonCheckAttack()
  */
 void SP_monster_demon1()
 {
-	if (deathmatch)
+	if (!KSPAWN_MONSTER(self))
```

Comment:
Guess it could be `if (deathmatch && !KSPAWN_MONSTER(self))` ... but that should have the same effect right?

Reply:
`deathmatch && !KSPAWN_MONSTER(self)` is better IMO, since I have no idea why !KSPAWN_MONSTER(self) implicitly imply deathmatch.